src/App/System/Controller/Config/Save.php

```diff
@@ -59,6 +59,9 @@ public function execute()
 
 		$application->enqueueMessage(g11n3t('The configuration file has been saved.'), 'success')
 			->redirect('/');
+
+		// To silence PHPCS expecting a return
+		return '';
```

Comment:
`Application::redirect()` should return some string :tongue:

Reply:
Personally, I'd rather deprecate `AbstractWebApplication::redirect()` and force controllers to set a response that triggers a redirect, similar to https://github.com/joomla/framework.joomla.org/blob/package-docs/src/Controller/Documentation/RedirectController.php, but getting that change accepted in the CMS ecosystem has odds somewhere around slim to none (as is the redirect method means you have to put extra logic in your code to either deal with sniffers like PHPCS warning about a missing return or add some kind of extra logic in the method for when tests trigger redirect and don't call `exit(0)` like the real method does).

Reply:
I like Symfonys way of returning response objects in controller methods, just return a `RedirectResponse` and let the application handle it..

Reply:
That's how I'm doing Framework apps now (well, actually it's more like `$app->setResponse()` and `return true;` because the controller interface defines a boolean return, `¯\_(ツ)_/¯`).  Unfortunately our controllers copied a lot of the glued together logic of the CMS' component helper and view classes setting up an output buffer and views echoing out stuff versus returning it.  And well, fixing this app to do Response objects is no small task given the number of controllers in place now.